src/server/db_slice.h

```diff
@@ -592,14 +592,20 @@ class DbSlice {
 
   DbTableArray db_arr_;
 
+  struct FpHasher {
+    size_t operator()(uint64_t val) const {
+      return val;
+    }
+  };
+
   // Used in temporary computations in Acquire/Release.
-  mutable absl::flat_hash_set<uint64_t> uniq_fps_;
+  mutable absl::flat_hash_set<uint64_t, FpHasher> uniq_fps_;
 
   // ordered from the smallest to largest version.
   std::list<std::pair<uint64_t, ChangeCallback>> change_cb_;
 
   // Used in temporary computations in Find item and CbFinish
-  mutable absl::flat_hash_set<CompactObjectView> fetched_items_;
+  mutable absl::flat_hash_set<uint64_t, FpHasher> fetched_items_;
```

Comment:
lets add a comment why we need this hashset , why we can not bumpup item twice in transaction without preemption but ok to to bumpup twice when we do preempt

Reply:
expect this comment the changes looks ok and we can merge